UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/NaturalChemicalReleaseSystem.cs

```diff
@@ -134,7 +134,7 @@ public void ToxinGeneration(float HeartEfficiency)
 			float Multiplier = HeartEfficiency;
 			if (HeartEfficiency == 0)
```

Comment:
this doesn't say "dead" specifically. Is there any situation where you have a bad heart but are still alive?

Reply:
You can be alive with no heart but None of the systems work such as Oxygenation

Reply:
would it be better to be more specific and check if the creature is dead instead?